### PR TITLE
xtest46: remove dependency on std.stdio

### PR DESCRIPTION
**Pull request by WalterBright**
Try to remove unnecessary Phobos dependencies from test suite.

### CODE REVIEW

File: test/runnable/xtest46.d [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // PERMUTE_ARGS: -unittest -O -release -inline -fPIC -g
 
-import std.stdio;
+//import std.stdio;
```
Comment (JinShil) [inline test/runnable/xtest46.d]: Just delete it, if it's not needed.

**Review by JinShil — APPROVED** [review]

File: test/runnable/xtest46.d [hunk 2]
```diff
@@ -2020,8 +2021,8 @@ void test96()
 {
     S96!([12, 3]) s1;
     S96!([1, 23]) s2;
-    writeln(s1.content);
-    writeln(s2.content);
+    //writeln(s1.content);
+    //writeln(s2.content);
```
Comment (JinShil) [inline test/runnable/xtest46.d]: Just delete them, if they're not needed

File: test/runnable/xtest46.d [hunk 3]
```diff
@@ -2213,8 +2214,8 @@ pure int genFactorials(int n) {
 void test107()
 {
     int[6] a;
-    writeln(a);
-    writeln(a.init);
+    //writeln(a);
+    //writeln(a.init);
```
Comment (JinShil) [inline test/runnable/xtest46.d]: Just delete them, if they're not needed.

File: test/runnable/xtest46.d [hunk 4]
```diff
@@ -3019,15 +3020,15 @@ struct Perm {
...
+            printf("%d\n", i);  // Never gets incremented.  Stays at 0.
         }
     }
 }
 
 void test136() {
     byte[] stuff = [0, 1, 2];
     auto perm2 = Perm(stuff);
-    writeln(perm2.perm);  // Prints [2, 0, 0]
+    //writeln(perm2.perm);  // Prints [2, 0, 0]
```
Comment (JinShil) [inline test/runnable/xtest46.d]: Just delete it, if it's not needed.

**Review by JinShil — APPROVED** [review]

File: test/runnable/xtest46.d [hunk 5]
```diff
@@ -3019,15 +3020,15 @@ struct Perm {
         foreach(elem; input) {
             enforce136(i < 3);
             perm[i++] = elem;
-            std.stdio.stderr.writeln(i);  // Never gets incremented.  Stays at 0.
+            printf("%d\n", i);  // Never gets incremented.  Stays at 0.
```
Comment (JinShil) [inline test/runnable/xtest46.d]: `fprintf()` to `stderr`?  Does it matter?

Comment (WalterBright) [reply test/runnable/xtest46.d]: no

File: test/runnable/xtest46.d (hunk 4 above)
Comment (WalterBright) [reply test/runnable/xtest46.d]: I find it convenient to comment/uncomment additional debug printing.

**Comment by JinShil** [comment]
Why does the status of the auto-tester appear to be stuck at "Pending: 10" for this PR? @braddr

**Pull request merged**